emcee/ptsampler.py

```diff
@@ -165,6 +164,10 @@ def __init__(self, ntemps, nwalkers, dim, logl, logp, threads=1,
         self.nwalkers = nwalkers
         self.dim = dim
 
+        # This is a random number generator that we can easily set the state
+        # of without affecting the numpy-wide generator
+        self._random = np.random.mtrand.RandomState()
+
```

Comment:
This duplicates code from `sampler.Sampler.__init__`
Not sure if that's a maintenance issue ...